driver-sync/src/test/functional/com/mongodb/client/AbstractRetryableWritesTest.java

```diff
@@ -254,7 +253,7 @@ public static Collection<Object[]> data() throws URISyntaxException, IOException
                 data.add(new Object[]{file.getName(), test.asDocument().getString("description").getValue(),
                         testDocument.getString("database_name", new BsonString(getDefaultDatabaseName())).getValue(),
                         testDocument.getArray("data"), test.asDocument(),
-                        !isDiscoverableReplicaSet() || skipTest(testDocument, test.asDocument())});
```

Comment:
I couldn't figure out why this was ever necessary, but it seems like it no longer is.

Reply:
Maybe the reason is because the tests fail on 4.2 and 4.4 sharded clusters.  Will look into that.

Reply:
I "fixed" it by bumping minServerVersion for sharded clusters to 5.0.  I'm going to check with other teams to see how they are passing these tests.

Reply:
Opened JAVA-5125 to find the root cause of the test failures, so I think we can proceed with this PR as it does increase the test coverage.